README.md

```diff
@@ -64,6 +64,8 @@ The UI Platform is developed as an isomorphic React application. View the list o
          ```bash
          oc whoami -t
          ```
+    
+    Note: If you want to develop against both the grc-ui & the grc-ui-api you can start the API server and configure the grc-ui to query against the local grc-ui-api server. See: [`grc-ui-api`](https://github.com/open-cluster-management/grc-ui-api)
```

Comment:
I think we should mention here that after sourcing the `.env` file, you can simply run `unset grcUiApiUrl` and it will default to the default local URL if you want to run grc-up-api locally.